mautic/mautic
# fix: list groups

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #14355 



## Description

This PR fixes situations where a weird black line appeared in several places.

example:
![image](https://github.com/user-attachments/assets/36793c09-6750-4103-8a7f-e45ce1c0031d)

Mautic was built using some Bootstrap elements in a random way
Some tabs use list groups, some tabs use navs and some use buttons
My changes rely on the class "list-group-tabs" to apply a different style to list groups that act as navigation, while restoring other list groups with the same appearance they had in July 5th this year.


**Instances (normal)**
- Variant list when viewing emails
![image](https://github.com/user-attachments/assets/095467d7-ebb0-46d2-9e1f-036103f6b3c9)

- Import dashboard
![image](https://github.com/user-attachments/assets/3fb3b579-1fee-41b3-8234-d99ff1b8dbbe)

- Viewing field from a form view
![image](https://github.com/user-attachments/assets/50f24c16-0e27-45a4-8b39-21e678338998)

- Selecting focus item purpose
![image](https://github.com/user-attachments/assets/ca9c2f1c-96de-477f-96d8-0b76dc3d7766)

- Selecting focus item format
![image](https://github.com/user-attachments/assets/517676c5-4286-4669-8576-6bb2f321a00a)

- Upcoming emails widget on dashboard
![image](https://github.com/user-attachments/assets/44e129e6-d126-40a6-8a6b-dbdcf6a1a816)


**Instances (tabs)**
- Browsing assets from remote storage
![image](https://github.com/user-attachments/assets/329dd3b3-f456-47c1-81b7-e990f6f5730b)

- Configuration page
![image](https://github.com/user-attachments/assets/778e443f-c53c-49a7-8e8f-0e85911f4402)

- System info page
![image](https://github.com/user-attachments/assets/13f9dbf0-9ef9-453c-89d5-82b3fc7bb235)

- Account page
![image](https://github.com/user-attachments/assets/37ff9a9a-bde6-4815-8fc1-38a69aaa6e59)

- Creating a company
![image](https://github.com/user-attachments/assets/0b0efc51-a6eb-470e-8ba6-5d3002d3d029)

- Creating a contact
![image](https://github.com/user-attachments/assets/e20edd9f-beaa-4fac-a76f-88479fa74edc)

- Creating role
![image](https://github.com/user-attachments/assets/978eaf20-59c9-4deb-90ff-c08644a7bf61)








---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create an A/B test for any email then check in its details the Variants tab
3. Go to the dashboard and use the import option on the dropdown
4. Create a form with at least 2 fields, save, open its details and check the fields tab
5. Open the focus item builder and check the first two tabs to select goal and format
6. See #14336 and follow its steps to see the upcoming emails widget
7. Open Plugins, click install, open Assets, create new, set Storage to Remote, click the Browse button and see
8. Open these pages: Configuration, System info, Account, New contact, New company, New role and see the list group
